install/install.sh

```diff
@@ -80,11 +81,14 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
     done
 fi
-supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >>  "$install_log" 2>&1
+#e=`mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password '${db_user_password}' >> "$install_log" 2>&1`
+#echo $e;
+mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password ;
```

Comment:
And store that newly entered password where exactly? It's needed further in the install script.